src/Plugin/WebformElement/CivicrmSelect.php

```diff
@@ -105,6 +105,14 @@ public function form(array $form, FormStateInterface $form_state) {
       '#prefix' => '<div id="webform-civicrm-options-wrapper">',
       '#suffix' => '</div>',
     ];
+    // M.H added this, provide empty option for CiviCRM Select
```

Comment:
This bit is separate from the rest of the code for this PR. This portion was to enable having an "Empty" option for elements that use this CivicrmSelect plugin, in the case that inspired this, Membership type. We needed a way to have an option in the select list that was '-Select-' .. so that when the form loads no membership type would be set as default, and we make the element required, forcing the user to make a choice for which membership type they want to purchase.